test/jdk/java/util/zip/ZipFile/EndOfCenValidation.java

```diff
@@ -160,6 +163,137 @@ public void shouldRejectInvalidCenOffset() throws IOException {
+         * 000039  z64 size           0
+         * 000047  z64 csize          0
+         *
+         * ------  File Data  ------
+         * 000055  data               7 bytes
+         *
+         * ------  Data Descriptor  ------
+         * 000062  signature          0x08074b50
+         * 000066  crc                0x3610a686
+         * 000070  csize              7
+         * 000078  size               5
+         *
+         * ------  Central Directory File Header  ------
+         * 000086  signature          0x02014b50
+         * 000090  made by version    45
+         * 000092  extract version    45
+         * 000094  flags              0x0808
+         * 000096  method             8              Deflated
+         * 000098  time               0x542c         10:33:24
+         * 000100  date               0x5947         2024-10-07
+         * 000102  crc                0x3610a686
+         * 000106  csize              4294967295
+         * 000110  size               4294967295
+         * 000114  diskstart          65535
+         * 000116  nlen               5
+         * 000118  elen               32
+         * 000120  clen               9
+         * 000122  iattr              0x00
+         * 000124  eattr              0x0000
+         * 000128  loc offset         4294967295
+         * 000132  name               5 bytes        'entry'
+         * 000137  ext id             0x0001         Zip64 extended information extra field
+         * 000139  ext size           28
+         * 000141  z64 size           5
+         * 000149  z64 csize          7
+         * 000157  z64 locoff         0
+         * 000165  z64 diskStart      0
+         * 000169  comment            9 bytes        'A comment'
+         *
+         * ------  Zip64 End of Central Directory Record  ------
+         * 000178  signature          0x06064b50
+         * 000182  record size        44
+         * 000190  made by version    45
+         * 000192  extract version    45
+         * 000194  this disk          0
+         * 000198  cen disk           0
+         * 000202  entries            1
+         * 000210  total entries      1
+         * 000218  cen size           92
+         * 000226  cen offset         86
+         *
+         * ------  Zip64 End of Central Directory Locator  ------
+         * 000234  signature          0x07064b50
+         * 000238  eoc disk           0
+         * 000242  eoc offset         178
+         * 000250  total disks        1
+         *
+         * ------  End of Central Directory  ------
+         * 000254  signature          0x06054b50
+         * 000258  this disk          0
+         * 000260  cen disk           0
+         * 000262  entries disk       65535
+         * 000264  entries total      65535
+         * 000266  cen size           4294967295
+         * 000270  cen offset         4294967295
+         * 000274  clen               0
+         */
+
+        byte[] zipBytes = HexFormat.of().parseHex("""
```

Comment:
It might be beneficial to future readers to provide the steps used to create the  hex string in the format you have provided.